src/huggingface_hub/_snapshot_download.py

```diff
@@ -379,12 +379,56 @@ def snapshot_download(
 
     results: List[Union[str, DryRunFileInfo]] = []
 
+    # User can use its own tqdm class or the default one from `huggingface_hub.utils`
+    tqdm_class = tqdm_class or hf_tqdm
+
+    # Create a progress bar for the bytes downloaded
+    # This progress bar is shared across threads/files and gets updated each time we fetch
+    # metadata for a file.
+    bytes_progress = tqdm_class(
+        desc="Downloading (incomplete total...)",
```

Comment:
(nit) maybe a clearer wording?  
```suggestion
        desc="Downloading (total size unknown...)",
```